### PR TITLE
React1 week2

### PR DESCRIPTION
**Pull request by oyelajakenny**
Implemented the following

Created a <NavItem> component,
Created a new List for the Pages 
Created a <SocialMediaItem /> component and replace all of the list items! 
Displayed the number Of selected planets 
Display the "no planets" message if it is empty!
 Added all 4 planets! Europa, Moon, Mars, Titan 
 Created a <PlanetCard /> component, which accepts the following properties: 
  name, description, thumbnail, isSelected, onAddOrRemovePlanet

### CODE REVIEW

File: app/destination/page.js [hunk 1]
```diff
@@ -1,40 +1,86 @@
...
+      thumbnail: "/destination/image-titan.png",
+    },
+  ];
+  const [selectedPlanets, setSelectedPlanets] = useState([]);
 
   let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  let numberOfPlanets = selectedPlanets.length;
 
   const onAddOrRemovePlanet = (name, index) => {
```
Comment (vdaguenet) [inline app/destination/page.js]:
```suggestion
  const onAddOrRemovePlanet = (name) => {
```
You don't use the `index` in your function

File: components/ui/Socialmedia.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import styles from "@/components/ui/Footer.module.css";
+import React from "react";
+
+const SocialLinks = ({ url, title, icon }) => {
```
Comment (vdaguenet) [inline components/ui/Socialmedia.js]:
```suggestion
const SocialLink = ({ url, title, icon }) => {
```
Keep the plural names for components containing multiple items.

File: components/ui/Navbar.js [hunk 3]
```diff
@@ -1,32 +1,59 @@
...
+  );
+};
+
+export default NavItem;
 
 export const Navbar = () => {
-  const currentPath = usePathname()
+  const [activeLink, setActiveLink] = useState();
+
+  const NavBarItems = [
```
Comment (vdaguenet) [inline components/ui/Navbar.js]:
```suggestion
  const navBarItems = [
```
PascalCase is reserved for components in React.

File: components/ui/Navbar.js [hunk 4]
```diff
@@ -35,25 +62,19 @@ export const Navbar = () => {
...
-            [styles.isLinkActive]: navbarItems[1].link === currentPath,
-          })}>
-            <Link href={navbarItems[1].link}><b>02</b> {navbarItems[1].title}</Link>
-          </li>
-          <li className={classNames(styles.navbarLinks, {
-            [styles.isLinkActive]: navbarItems[2].link === currentPath,
-          })}>
-            <Link href={navbarItems[2].link}><b>03</b> NASA COLLABORATION</Link>
-          </li>
+          {NavBarItems.map((NavItems, index) => (
```
Comment (vdaguenet) [inline components/ui/Navbar.js]:
```suggestion
          {NavBarItems.map((navItems, index) => (
```
Same comment about casing.

**Review by vdaguenet — APPROVED** [review]
Cool!

**Comment by oyelajakenny** [comment]
Thank you for your Feedback @vdaguenet

**Pull request merged**